libs/application/templates/children-residence-change/src/fields/ParentBIntro/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { useIntl } from 'react-intl'
+import { Box, Button, Text } from '@island.is/island-ui/core'
+
+import {
+  childrenResidenceInfo,
+  formatAddress,
+  getSelectedChildrenFromExternalData,
+} from '../../lib/utils'
+import { parentBIntro } from '../../lib/messages'
+import { CRCFieldBaseProps } from '../../types'
+import { DescriptionText } from '../components'
+
+const ParentBIntro = ({ field, application }: CRCFieldBaseProps) => {
+  const { externalData, answers } = application
+  const { formatMessage } = useIntl()
+  const applicant = externalData.nationalRegistry.data
+  const childResidenceInfo = childrenResidenceInfo(applicant, answers)
+  const children = getSelectedChildrenFromExternalData(
+    applicant.children,
+    answers.selectedChildren,
+  )
+  return (
+    <>
+      <Box marginTop={3} marginBottom={5}>
+        <Text>
+          {formatMessage(parentBIntro.general.description, {
+            otherParentName: applicant.fullName,
+          })}
+        </Text>
+      </Box>
+      <Box marginTop={4} marginBottom={4}>
+        <Text variant="h4" marginBottom={1}>
+          {formatMessage(parentBIntro.information.childNameTitle, {
+            count: children.length,
+          })}
+        </Text>
+        {children.map((child) => (
+          <Text key={child.nationalId}>{child.fullName}</Text>
+        ))}
+      </Box>
+      <Box marginBottom={4}>
+        <Text variant="h4">
+          {formatMessage(parentBIntro.information.currentLegalParentHomeLabel)}
+        </Text>
+        <Text variant="h4" color="blue400">
+          {childResidenceInfo.current.parentName}
+        </Text>
+        <Text fontWeight="light">
+          {formatAddress(childResidenceInfo.current.address)}
+        </Text>
+      </Box>
+      <Box marginBottom={5}>
+        <Text variant="h4">
+          {formatMessage(parentBIntro.information.newLegalParentHomeLabel)}
+        </Text>
+        <Text variant="h4" color="blue400">
+          {childResidenceInfo.future.parentName}
+        </Text>
+        <Text fontWeight="light">
+          {formatAddress(childResidenceInfo.future.address)}
+        </Text>
+      </Box>
+      <Text variant="h4">{formatMessage(parentBIntro.disagreement.title)}</Text>
+      <Box marginTop={1} marginBottom={2}>
```

Comment:
Same here, I would prefer to have `marginBottom` on the `<Text>` component and only `marginBottom` on Box

Reply:
I also think the margin bottom should be set to 5 instead of 2 here? 40px instead of 16px